File: web/src/api/yunguan/config/alertConfig.js
import request from '@/utils/request'
// 添加元素
export function addAlertConfig(modelData) {
    return request({
        url: 'config/alertConfig/addAlertConfig',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function updateAlertConfig(modelData) {
    return request({
        url: 'config/alertConfig/updateAlertConfig',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function deleteAlertConfig(modelData) {
    return request({
        url: 'config/alertConfig/deleteAlertConfig',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function queryAlertConfig(modelData) {
    return request({
        url: 'config/alertConfig/queryAlertConfig',
        method: 'post',
        data: modelData
    })
}

File: server/router/yunguan/config/alertConfig.go
package config

import (
	"github.com/gin-gonic/gin"
	v1 "ygpt/server/api/v1"
)

type AlertConfigRouter struct {
}

func (s *LicenseConfigRouter) InitAlertConfigRouter(Router *gin.RouterGroup) {
	dataRouter := Router.Group("config/alertConfig")
	backApi := v1.ApiGroupApp.YunGuanApiGroup.ConfigApi.AlertConfigApi

	{
		dataRouter.POST("addAlertConfig", backApi.AddAlertConfig)       // 添加数据
		dataRouter.POST("updateAlertConfig", backApi.UpdateAlertConfig) // 修改数据
		dataRouter.POST("deleteAlertConfig", backApi.DeleteAlertConfig) // 修改数据
		dataRouter.POST("queryAlertConfig", backApi.QueryAlertConfig)   // 修改数据
	}
}
